Vagrantfile

```diff
@@ -1,15 +1,17 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 Vagrant.configure('2') do |config|
-  config.vm.box      = 'ubuntu/focal64' # 20.04
-  config.vm.hostname = 'rails-dev-box'
+  config.vm.box      = 'ubuntu/bionic64' # 18.04
+  config.vm.hostname = 'activerecord-sqlserver-adapter-dev-box'
 
   config.vm.network :forwarded_port, guest: 3000, host: 3000
+  config.vm.network :forwarded_port, guest: 1433, host: 1433
 
   config.vm.provision :shell, path: 'bootstrap.sh', keep_color: true
+  config.vm.provision :shell, path: 'bootstrap_sqlserver.sh', keep_color: true
 
   config.vm.provider 'virtualbox' do |v|
-    v.memory = ENV.fetch('RAILS_DEV_BOX_RAM', 2048).to_i
-    v.cpus   = ENV.fetch('RAILS_DEV_BOX_CPUS', 2).to_i
+    v.memory = ENV.fetch('ACTIVERECORD_SQLSERVER_DEV_BOX_RAM', 4098).to_i
```

Comment:
Can you add the workaround described in https://github.com/hashicorp/vagrant/issues/9524 to the VM config? Not sure if that happens to everyone, but I got that error on Ubuntu.

`v.customize ['modifyvm', :id,  '--audio',  'none']`

Or adding a proper name to the VM also does the trick:

`v.name   = 'activerecord-sql-server-dev-box'`

Reply:
Added `v.name = 'activerecord-sql-server-dev-box'`.